### DISCUSSION TITLE
Is it possible to freeze some layers while training?

### DISCUSSION THREAD

**Discussion by leon-maechler** [FETS-AI/Challenge#78] [Q&A] [answered]
Hello!

As far as I can tell, it is not possible to freeze some layers while training (i.e. have them not change values). I just wanted to confirm this quickly to make sure. 

Thanks!

**Comment by sarthakpati** [comment, marked as answer]
Hi,

That is indeed correct. Model-level changes are not allowed in this iteration of the FeTS challenge.

Cheers,
Sarthak

**Reply by leon-maechler** [reply]
Ok thanks!
